.github/workflows/translations-sync.yml

```diff
@@ -0,0 +1,36 @@
+name: Sync translations
+
+on:
+  workflow_dispatch:
+  push:
+    paths: ['bridge_ui/locales/**']
+    branches: [master]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync-translations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Sync sources and translations
+        uses: crowdin/github-action@v2
+        with:
+          config: 'crowdin.yml'
+          upload_sources: true
+          upload_translations: true
+          download_translations: true
+          export_only_approved: true
+          localization_branch_name: l10n_crowdin_translations
+          create_pull_request: true
+          pull_request_title: 'New Crowdin translations'
+          pull_request_body: 'New translations from Crowdin.'
+          pull_request_base_branch_name: 'master'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
+          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
```

Comment:
We need to set these 2 secrets up in this project. Hit me up to give you the secrets.